packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/ReminderSection/SelectFieldWithEdit.tsx

```diff
@@ -0,0 +1,78 @@
+import { ControllerRenderProps, FieldValues } from 'react-hook-form';
+
+import { RHFSelectField } from '@/components/RHF/RHFSelectField';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Button, Grid, MenuItem, SelectChangeEvent } from '@mui/material';
+
+import { ReminderDetail, ReminderDetails } from './ReminderDetail';
+
+interface CustomSelectFieldProps extends FieldValues {
```

Comment:
`extends FieldValues` in my opinion is not the correct way. Most of field such a `createOptionLabel` or `details` are just `any`.
<img width="592" alt="image" src="https://github.com/user-attachments/assets/19f58bc9-6ee8-4e73-b79f-141bfec17f92">

Reply:
Fixed, with correct typings